CrewChiefV4/Events/OvertakingAidsMonitor.cs

```diff
@@ -62,20 +64,28 @@ override protected void triggerInternal(GameStateData previousGameState, GameSta
                 }
                 if (currentGameState.SessionData.IsNewLap)
                 {
-                    if (drsAvailableOnThisLap && !hasUsedDrsOnThisLap)
+                    if (drsAvailableOnThisLap && !hasUsedDrsOnThisLap
+                        || (CrewChief.gameDefinition.gameEnum == GameEnum.RF2_64BIT && drsAvailableInAnyZoneOnThisLap && !hasUsedDrsOnThisLap))
```

Comment:
because I'd old and thick, can you bracket these conditions - should it be ((a && !b) || (c && d && !e))?

Reply:
will do.